src/vs/workbench/contrib/debug/browser/repl.ts

```diff
@@ -340,6 +339,7 @@ export class Repl extends ViewPane implements IHistoryNavigationWidget {
 					background-color: ${backgroundColor};
 				}
 			`;
+			this.styleElement.style.setProperty(`--vscode-repl-font-family`, fontFamily);
```

Comment:
I believe my original example was incorrect. Instead of calling `.style.setProperty` on the style node, you need to call it on a parent node of the repl node.

In the webview for example, we always call it on the top level document element: https://github.com/microsoft/vscode/blob/10514bf423ab0b765790889cf06f0b6e569254a7/src%2Fvs%2Fworkbench%2Fcontrib%2Fwebview%2Fbrowser%2Fpre%2Fmain.js#L215